charts/lws/README.md

```diff
@@ -24,16 +24,24 @@ Quick start instructions for the setup and configuration of lws using Helm.
 
 ##### Install chart using Helm v3.0+
 
+Either clone the lws repository:
+
+```bash
+git clone git@github.com:kubernetes-sigs/lws.git
+cd charts
+helm install lws lws --create-namespace --namespace lws-system
+```
+
+Or use the charts pushed to `oci://registry.k8s.io/lws/charts/lws`:
```

Comment:
###### Install a released version

Refer to helm chart installation [page](https://lws.sigs.k8s.io/docs/installation/#install-by-helm) for details.

Reply:
Thanks @kerthcet 

It has been changed :-)